src/Styleguide/Components/RelayPagination.tsx

```diff
@@ -0,0 +1,193 @@
+
+  return (
+    <Flex flexDirection="row">
+      {pageCursors.first && (
+        <div>
+          {renderPage(pageCursors.first, props.onClick)}
+          <PageSpan mx={0.5} />
+        </div>
+      )}
+
+      {pageCursors.around.map(pageInfo => renderPage(pageInfo, props.onClick))}
+
+      {pageCursors.last && (
+        <div>
+          <PageSpan mx={0.5} />
+          {renderPage(pageCursors.last, props.onClick)}
+        </div>
+      )}
+
+      <PrevButton onClick={() => props.onPrev()} />
+      <NextButton onClick={() => props.onNext()} />
+    </Flex>
+  )
+}
+
+export const SmallPagination = (props: PaginationProps) => {
+  return (
+    <Flex flexDirection="row" width="100%">
+      <Flex width="50%" pr={0.5}>
+        <ButtonWithBorder
+          alignItems="center"
+          justifyContent="flex-start"
+          pl={1}
+          onClick={() => props.onPrev()}
+        >
+          <Arrow direction="left" />
+        </ButtonWithBorder>
+      </Flex>
+      <Flex width="50%" pl={0.5}>
+        <ButtonWithBorder
+          onClick={() => props.onNext()}
+          alignItems="center"
+          justifyContent="flex-end"
+          pr={1}
+        >
+          <Arrow direction="right" />
+        </ButtonWithBorder>
+      </Flex>
+    </Flex>
+  )
+}
+
+const scrollToTop = clickHandler => {
+  return () => {
+    window.scrollTo(0, 0)
+    clickHandler()
+  }
+}
+
+const Page = ({ num, onClick, ...props }) => {
+  return (
+    <Button {...props} onClick={() => scrollToTop(onClick)()}>
+      <Sans size="3" weight="medium" display="inline">
+        {num}
+      </Sans>
+    </Button>
+  )
+}
+
+const PageSpan = ({ mx }) => {
+  return (
+    <Sans size="3" display="inline" mx={mx} color="black30">
+      ...
+    </Sans>
+  )
+}
+
+const PrevButton = props => {
+  return (
+    <Sans size="3" weight="medium" display="inline" mx={0.5}>
+      <a onClick={() => scrollToTop(props.onClick)()} className="noUnderline">
+        <Arrow direction="left" /> Prev
+      </a>
+    </Sans>
+  )
+}
+
+const NextButton = props => {
+  return (
+    <Sans size="3" weight="medium" display="inline" mx={0.5}>
+      <a onClick={() => scrollToTop(props.onClick)()} className="noUnderline">
+        Next <Arrow direction="right" />
+      </a>
+    </Sans>
+  )
+}
+
+const activeButton = css`
+  background: ${themeGet("colors.black5")};
+  border-radius: 2px;
+  border: 0;
+`
+
+const Button = styled.button.attrs<{ active?: boolean }>({})`
+  cursor: pointer;
+  width: 23px;
+  height: 25px;
+  background: transparent;
+  border: 0;
+
+  outline: 0;
+
+  ${p => p.active && activeButton};
+
+  &:hover {
+    ${activeButton};
+  }
+`
+
+const ButtonWithBorder = styled(Flex)`
+  border: ${props => props.theme.borders[1]};
+  border-color: ${themeGet("colors.black10")};
+  border-radius: 3px;
+  width: 100%;
+  height: ${props => props.theme.space[4]}px;
+  cursor: pointer;
+`
+
+export const RelayPagination = createFragmentContainer(
```

Comment:
Ahh, I missed this earlier. Are we abandoning the default export for the fragment containers?

Reply:
We're started avoiding default exports since they're not statically analyziable.